ibc-core/ics04-channel/types/src/channel.rs

```diff
@@ -279,11 +279,6 @@ impl ChannelEnd {
         Ok(())
     }
 
```

Comment:
```suggestion

    /// Helper function to compare the order of this end with another order.
    #[deprecated(
        since = "0.50.1",
        note = "Use `Eq` or `match` directly on the `Order` enum instead"
    )]
    pub fn order_matches(&self, other: &Order) -> bool {
        self.ordering.eq(other)
    }
```